cms/djangoapps/contentstore/rest_api/v2/views/downstreams.py

```diff
@@ -0,0 +1,251 @@
+"""
+API Views for managing & syncing links between upstream & downstream content
+
+API paths (We will move these into proper api_doc_tools annotations soon
+https://github.com/openedx/edx-platform/issues/35653):
+
+  /api/v2/contentstore/downstreams/{usage_key_string}
```

Comment:
Should this be `/api/contentstore/v2/downstreams/{usage_key_string}` here (and elsewhere in this docstring)?

Reply:
@ormsbee yup, will fix

Reply:
Done